File: uri/contest_SBC/g.cpp
#include<bits/stdc++.h>

using namespace std;

int main(){
    int n, v, total;
    float aux=0;
    
    cin >> v >> n;

    total = v * n;
    
    for(float i=0.1; i < 1; i += 0.1){
        aux = total * i;
        cout << "result: " << (aux - int(aux)) << endl;
        if(aux - (int(aux)) > 0.0){
            aux++;
        }

        if(i != 0.9) printf("%d ", int(aux));
        else  printf("%d\n", int(aux));
    }
    return 0;
}

File: uri/contest_SBC/g.py
import math as m

v,n = map(int, input().split())

total = v * n
ans = []

for i in range(1,10):
    aux = total * (i/10)
    aux = m.ceil(aux)
    ans.append(int(aux))

for i in range(len(ans)-1):
    print(ans[i], end=' ')
print(ans[-1])